controllers/space/space_controller.go

```diff
@@ -340,7 +340,10 @@ func (r *Reconciler) manageNSTemplateSet(logger logr.Logger, space *toolchainv1a
 			}
 			nsTmplSet = NewNSTemplateSet(memberCluster.OperatorNamespace, space, spaceBindings.Items, tmplTier)
 			if err := memberCluster.Client.Create(context.TODO(), nsTmplSet); err != nil {
-				logger.Error(err, "failed to create NSTemplateSet on target member cluster")
+				if errors.IsAlreadyExists(err) {
+					// requeue immediately, there's probably a race condition between the host client cache and the member cluster state
+					return nsTmplSet, requeueDelay, nil
```

Comment:
you wrote 
> // requeue immediately

but 
```
requeueDelay = 1 * time.Second
```

Reply:
granted, the wording was confusing so I removed the "immediately" word in the sentence in 26423d6